packages/lesswrong/lib/collections/emailTokens/schema.ts

```diff
@@ -15,7 +15,7 @@ const schema = {
       type: "User",
     })
   },
-  used: {
+  usedAt: {
```

Comment:
Isn't this going to break stuff?

Reply:
Oddly enough, no; `emailTokens` was being used (non-typechecked) as though the field were named `usedAt` already; the schema was failing to reflect the usages. Since there's already data in the database, it's easier to change the schema to reflect the code than vise versa.